reccmp/tools/vtable.py

```diff
@@ -68,9 +68,16 @@ def main():
         logger.error(e.args[0])
         return 1
 
-    with IsleBin(target.original_path) as orig_bin, IsleBin(
-        target.recompiled_path
-    ) as recomp_bin:
+    orig_bin = detect_image(args.original)
+    assert isinstance(orig_bin, PEImage)
+
+    recomp_bin = detect_image(args.recompiled)
+    assert isinstance(recomp_bin, PEImage)
+    engine = IsleCompare(orig_bin, recomp_bin, args.pdb, args.decomp_dir)
+
+    # FIXME: remove "if True"
```

Comment:
thanks, this greatly increases the comparability. Feel free to remove it after @disinvite has taken another look